docs/uefi.md

```diff
@@ -0,0 +1,42 @@
+# UEFI Boot
+
+Cloud Hypervisor supports UEFI boot through the utilization of the EDK II based UEFI firmware. 
+
+## Building UEFI Firmware
+
+To avoid any unnecessary issues, it is recommended to use Ubuntu 18.04 and its default toolset. Any other compatible Linux distribution is otherwise suitable, however it suggests itself to use a temporary docker container with Ubuntu 18.04 for a quick worry-free build on some existing Linux machine.
```

Comment:
`To avoid any unnecessary issues, it is recommended to use Ubuntu 18.04 and its default toolset. Any other compatible Linux distribution is otherwise suitable, however it is suggested to use a temporary docker container with Ubuntu 18.04 for a quick worry-free build on some existing Linux machine.`